platform/tests/TESTS/mbed_functional/callback/main.cpp

```diff
@@ -885,9 +887,10 @@ Case cases[] = {
     Case("Testing callbacks with 3 uint64s", test_dispatch3<uint64_t>),
     Case("Testing callbacks with 4 uint64s", test_dispatch4<uint64_t>),
 // IAR currently crashes at link time with this test - skip it as it's well beyond anything needed by real code
```

Comment:
Should we update this comment to reflect the current reason for skipping, or just delete it? The commit message explains why we're doing this, so I don't see a need for this out of date comment.